lib/typing/values/string.go

```diff
@@ -57,22 +57,23 @@ func ToString(colVal any, colKind typing.KindDetails, additionalDateFmts []strin
 
 		return stringutil.EscapeBackslashes(fmt.Sprint(colVal)), nil
 	case typing.Struct.Kind:
-		if colKind == typing.Struct {
-			if strings.Contains(fmt.Sprint(colVal), constants.ToastUnavailableValuePlaceholder) {
-				colVal = map[string]any{
-					"key": constants.ToastUnavailableValuePlaceholder,
-				}
-			}
-
-			if reflect.TypeOf(colVal).Kind() != reflect.String {
-				colValBytes, err := json.Marshal(colVal)
-				if err != nil {
-					return "", err
-				}
+		if strings.Contains(fmt.Sprint(colVal), constants.ToastUnavailableValuePlaceholder) {
```

Comment:
No need to create a map just to have it marshaled, we can do it ourselves. Also the if statement is a no-op given the case.